### PR TITLE
[bitnami/fluentd] Add missing features from stable/fluentd

### PR DESCRIPTION
**Pull request by alemorcuq**
**Description of the change**
The goal of this PR is to add the missing features from the `stable/fluentd` chart for a smoother transition from that chart. See this thread for a discussion on this: https://github.com/helm/charts/issues/21235#issuecomment-721561451

Missing features implemented:
- [x] Allow Fluentd configuration without having to use an external ConfigMap.
- [x] Use a HPA for the StatefulSet (interesting in aggregator-only deployments).
- [x] Add an Ingress for the aggregator

**Checklist**
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by alemorcuq** [comment]
Sorry for the delay, this is ready for review now.

File: bitnami/fluentd/README.md [hunk 1]
```diff
@@ -141,6 +144,20 @@ The following tables lists the configurable parameters of the fluentd chart and
...
+| `aggregator.ingress.enabled`                    | Enable ingress controller resource                                                                             | `false`                                                                                                 |
+| `aggregator.ingress.certManager`                | Add annotations for cert-manager                                                                               | `false`                                                                                                 |
+| `aggregator.ingress.hostname`                   | Default host for the ingress resource                                                                          | `wordpress.local`                                                                                       |
+| `aggregator.ingress.path`                       | Default path for the ingress resource                                                                          | `/`                                                                                                     |
```
Comment (FraPazGal) [inline bitnami/fluentd/README.md:150]: Probably a typo, you forgot to add the parameter `aggregator.ingress.pathType` here.

File: bitnami/fluentd/values-production.yaml [hunk 2]
```diff
@@ -443,6 +616,90 @@ aggregator:
...
+    ## If you're using cert-manager, this is unneeded, as it will create the secret for you if it is not set
+    ##
+    ## It is also possible to create and manage the certificates outside of this helm chart
+    ## Please see README.md for more information
+    ##
+    secrets: []
+    ## - name: fluentd.local-tls
+    ##   key:
+    ##   certificate:
+    ##
```
Comment (FraPazGal) [inline bitnami/fluentd/values-production.yaml:701]: I think there is some configuration missing. You are not creating the secrets from this input.

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Thank for the PR, these are good additions to make the migration easier! Could you please take a look at my comments?

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**